index.html

```diff
@@ -4,7 +4,17 @@
 		<meta charset="UTF-8" />
 		<link rel="icon" type="image/svg+xml" href="/vite.svg" />
 		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
-		<title>Vite + React + TS</title>
+		<meta name="twitter:card" content="summary_large_image" />
+		<meta name="twitter:title" content="CommunityFocus" />
+		<meta name="twitter:description" content="Create a sharable timer for your team or friends!" />
+		<meta name="twitter:image" content="https://i.imgur.com/gPkVD1Q.png" />
+
+		<meta property="og:title" content="CommunityFocus" />
+		<meta property="og:description" content="Create a sharable timer for your team or friends!" />
+		<meta property="og:image" content="https://i.imgur.com/gPkVD1Q.png" />
+		<meta property="og:url" content="https://www.CommunityFocus.app" />
+		<meta property="og:type" content="website" />
+		<title>CommunityFocus</title>
```

Comment:
```suggestion
		<title>Vite + React + TS</title>
```

In order to stick to the scope of this ticket, can we revert this and not do work outside of the requirements in the ticket?
This way, that change can be taken by someone else (even if its minor) so that learn how to push via github.